configs/model/contrastive/vicreg.yaml

```diff
@@ -0,0 +1,44 @@
+_target_: cyto_dl.models.contrastive.Contrastive
+
+anchor_key: image
+positive_key: image_aug
+target_key: target
```

Comment:
what is the "target" key? Didn't see this in the data loader

Reply:
the mnist dataloader pops out that the label as well